notebooks/demos/2022-12/SKEMA-Demo.ipynb

```diff
@@ -0,0 +1,65 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "28fbceab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "4d6dcb85",
+   "metadata": {},
+   "outputs": [
```

Comment:
@enoriega  - I think it would be best to not commit Jupyter notebook outputs to the repo. Would you be able to clear the outputs and recommit the notebooks (SKEMA-Demo.ipynb and skema_text_reading_demo.ipynb)?

Reply:
You're right. Sorry!

Reply:
No problem, thanks for the PR!